app/(unlogin)/page.tsx

```
@@ -0,0 +1,121 @@
+import { Fragment } from "react";
+import Link from "next/link";
+import "./index.css";
+
+// from date.strftime('%Y.%m.%d')
+// TODO: use dayjs
+function formatDate(date: Date) {
+  return (
+    date.getFullYear() +
+    "." +
+    String(date.getMonth() + 1).padStart(2, "0") +
+    "." +
+    String(date.getDate()).padStart(2, "0")
+  );
+}
+
+export default async function Index() {
+  // TODO: get data from database
+  const main_content = <></>;
+  const tags = [
+    { no: 1, content: "tag1" },
+    { no: 2, content: "tag2" },
+  ];
+  const workshop_list = [
+    {
+      no: 0,
+      nth: 21,
+      start_date: new Date(),
```

여긴 아직 mock data긴 하지만... 렌더할 때마다 값이 달라지는 impure한 부분이 있으면 hydration error를 밟기 쉽습니다. 회피 방법은 몇 가지가 있는데 아무튼 앞으로 생각해보면 좋을 것 같아요.